contracts/child/MRC20.sol

```diff
@@ -103,6 +103,7 @@ contract MRC20 is BaseERC20 {
     function _transfer(address sender, address recipient, uint256 amount)
         internal
     {
+        require(recipient != address(this), "can't send to MRC20");
```

Comment:
why this?

Reply:
Just in case, to avoid stuck eth in the contract.